### DISCUSSION TITLE
How to set alias in absolute URL

### DISCUSSION THREAD

**Discussion by ateliee** [nuxt/nuxt#17233] [Questions] [answered]
I want to set an alias with an absolute URL in another domain in Nuxt3(rc-6).

.env
``` 
STATIC_ABSOLUTE_URL=https://example.com
```

nuxt.config.ts
```
export default defineNuxtConfig({
    alias: {
         '~static': process.env.STATIC_ABSOLUTE_URL
    }
});
```

app.vue
```
.main{
     background-image: url("~static/sample.png");
}
```

Probably due to the effect of file-loader or an error. Is there a way to deal with it?

```
[vite:css] ENOENT: no such file or directory, open 'https://example.com/sample.png'
```

**Comment by leonardssh** [comment, marked as answer]
Have you tried with [cdnURL](https://v3.nuxtjs.org/api/configuration/nuxt.config#cdnurl)?

**Reply by ateliee** [reply]
Thank you!
It was very helpful!
